doc/build-osx.md

```diff
@@ -43,10 +43,10 @@ NOTE: Building with Qt4 is still supported, however, could result in a broken UI
         git clone https://github.com/bitcoin/bitcoin.git
         cd bitcoin
 
-2.  Build bitcoind:
+2.  Build bitcoin-core:
 
```

Comment:
Nit: Add a line with sth. like: "This will configure and build the headless bitcoin binaries and the bitcoin gui (if qt is found). You can disable the gui build with `--disable-gui`"?